mantidproject/mantid
# Use the multislice view for single crystal diffraction

Description of work.

This changes the technique-dependent initial view for a MDHistoworkspace from a single crystal diffraction instrument to the the MultiSlice view. This avoids the very slow `vtkDataSetSurfaceFilter` that is causing unacceptably slow start times.

I made additional changes to avoid switching to standard view for rendering.

**To test:**



1. In the MD Plotting preferences, change the initial view back to "technique-dependendent". Restart MantidPlot. 
2. Open a MDHistoworkspace from a SCD instrument such as TOPAZ, view it in the VSI and verify that it opens in the MultSlice view.
3. Verify there are no issues switching between views.
4. Open a MDEventWorkspace, view it in the VSI and verify that the SplatterPlot opens correctly and there are no issues switching views. 

There is no GitHub issue associated with this pull request.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
